XCUITests/LibraryTests.swift

```diff
@@ -23,3 +23,39 @@ class LibraryTest: IpadOnlyTestCase {
         waitForExistence(app.tables["History List"])
     }
 }
+
+class LibraryTestsIphone: IphoneOnlyTestCase {
+    
+    func testLibraryShortcutHomePage () {
+        if skipPlatform {return}
+        waitForExistence(app.staticTexts["libraryTitle"])
+        waitForExistence(app.buttons["menu Bookmark"])
+        waitForExistence(app.buttons["menu panel History"])
+        waitForExistence(app.buttons["menu panel ReadingList"])
+        waitForExistence(app.buttons["menu panel Downloads"])
+        
+        // Check if clicking on Bookmark option shows bookmarks
+        app.buttons["menu Bookmark"].tap()
+        navigator.nowAt(HomePanel_Bookmarks)
+        waitForExistence(app.tables["Bookmarks List"])
+        navigator.goto(HomePanelsScreen)
+        
+        // Check if clicking on History option shows history
+        app.buttons["menu panel History"].tap()
+        navigator.nowAt(HomePanel_History)
+        waitForExistence(app.tables["History List"])
+        navigator.goto(HomePanelsScreen)
+        
+        // Check if clicking on Reading List option shows history
+        app.buttons["menu panel ReadingList"].tap()
+        navigator.nowAt(HomePanel_ReadingList)
+        waitForExistence(app.tables["HomePanels.ReadingList"])
```

Comment:
The test is failing on BB and for me locally too, I found that this id is not the correct one...instead this one should be used if I'm not wrong: `(app.tables["ReadingTable"])`